admin/src/pages/Order.jsx

```
@@ -0,0 +1,128 @@
+import { toast } from "react-toastify";
+import parcel_icon from "../assets/parcel.png";
+
+const Order = ({ token }) => {
+  const [orders, setOrders] = useState([]);
+
+  const fetchAllOrders = async () => {
+    if (!token) return null;
+
+    try {
+      const res = await axios.post(
+        backendUrl + "/api/order/list",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)